### ISSUE TITLE
[MS] Make minimum_deposit modifiable according to token price

### ISSUE THREAD

**Issue by czepluch** [raiden-network/raiden-contracts#75]
## Problem definition

In the Monitoring Service smart contract we want a minimum deposit to be made in order to register as a Monitoring Service. Right now the value is not modifiable and is specified when the contract is created. We need to come up with a good way to be able to adjust the `minimum_deposit` without already registered MSs having to deposit more tokens.

## Possible solution
Currently two approaches have been suggested:

1. Have an owner that can update the `minimum_deposit` on price fluctuation
2. Use an oracle to update the value when price is fluctuating

A third solution could also be to just hard code a value that reflects some fraction of the total token supply.

## Next steps
- [ ] Discuss what the best solution is
- [ ] Implement solution
- [ ] Update spec

**Comment by LefterisJP** [comment]
I don't like either of the 2 solutions.  Having an owner is out of the question. Oracle is not much better. Let's first discuss why do we need a minimum deposit to be made in the first place? What is the purpose of the deposit?

The people running an RSB would put up a [deposit](https://github.com/raiden-network/spec/issues/79) in order to show they can be trusted and to be held responsible in case of cheating. Then they would be listed in the registry of third party services in some order depending on that deposit. If the way to display and order these third party services is calculated relative to the deposit of other RSB operators why do we even need a minimum deposit?

**Comment by czepluch** [comment]
Makes sense. I can create a "mock" RSB smart contract to use in the MS smart contract then and entirely get rid of the `minimum_deposit` then.

**Comment by loredanacirstea** [comment]
@czepluch , shouldn't we still keep this open for the RSB contract? The issue is still valid.

**Comment by czepluch** [comment]
Maybe rather create a new issue about the RSB contract according to https://github.com/raiden-network/spec/issues/79, but we can also just reopen this and edit the issue.

**Comment by czepluch** [comment]
Ah, you edited your comment. I would rather open a new issue.
